site/app/templates/forum/createThread.twig

```diff
@@ -33,7 +33,7 @@
             "show_thread_status" : true,
             "show_announcement" : true,
             "show_editcat" : true,
-            "submit_label" : "Submit Post",
+            "submit_label" : "Publish thread",
```

Comment:
This line will break the e2e tests. Please update the e2e tests to search for `Publish thread`

Reply:
i updated it, see the last commit https://github.com/Submitty/Submitty/pull/5529/commits/1be42ff4f7ed1836398c24e6e96e90b7e0a1b6bc